devolutions-agent/src/updater/package.rs

```diff
@@ -87,9 +88,11 @@ fn ensure_enough_rights() -> Result<(), UpdaterError> {
         GetTokenInformation(
             *token_handle,
             TokenElevation,
-            Some(&mut token_elevation as *mut _ as _),
-            size_of::<TOKEN_ELEVATION>().try_into().unwrap(),
-            &mut return_size as _,
+            Some(&mut token_elevation as *mut TOKEN_ELEVATION as *mut core::ffi::c_void),
+            size_of::<TOKEN_ELEVATION>()
+                .try_into()
+                .expect("Unreachable: TOKEN_ELEVATION size always fits into u32"),
```

Comment:
style:
```suggestion
                .expect("TOKEN_ELEVATION size always fits into u32"),
```

Reply:
I think it's debatable -- triggering `expect` is always either some world-breaking bug, which happened because previous assumptions about the execution path were wrong, or just simply unreachable code path, which should never be reached, I like to prepend expect's with category like `BUG:` or `Unreachable:` to quickly categorize the problem source during log analysis/debug, but if it's deal breaker, I could remove this prefix 🙂

Reply:
I would argue that if you get a panic with a "Unreachable:" prefix, you have a "bug", and conversely we always expect the panic branch to be "unreachable" unless there is a bug. Then, sometimes the absence of bug is obvious, sometimes it’s not, but I don’t really see how it matters once the program is crashed. I think the `Unreachable:` prefix is making the already busy line even more busy for not much. If you’re still not convinced, feel free to leave that way!

Reply:
Makes sense 👍